workspaces/src/types/account.rs

```diff
@@ -393,7 +395,9 @@ pub struct AccountDetails {
     pub balance: NearToken,
     pub locked: NearToken,
     pub code_hash: CryptoHash,
-    pub storage_usage: u64,
+    pub storage_usage: StorageUsage,
+    // TODO: protocol_feature_nonrefundable_transfer_nep491 is not supported by near-cli-rs
```

Comment:
Is it TODO for near-workspaces-rs or for near-cli-rs?

Reply:
@akorchyn Protocol feature flags on nearcore creates are dropped after they get stabilized. Are you sure you need an unstable protocol feature here?